opengemini_client/client_impl.py

```diff
@@ -128,8 +131,25 @@ def ping(self, idx: int):
             return None
         return Error(f"ping openGeminiDB status is {resp.status_code}")
 
-    def query(self, query: Query) -> tuple:
-        return ()
+    def query(self, query: Query):
+
+        if query.command.lower().startswith("select") and "into" not in query.command.lower():
+            method = 'GET'
+        else:
+            method = 'POST'
+
+        server_url = self.get_server_url()
+        url_path = UrlConst.QUERY
```

Comment:
The variable 'url_path' is unnecessary and can be inlined.

Reply:
I has been inlined